third_party/move/move-compiler-v2/tests/lambda/storable/parse_errors.move

```diff
@@ -1,107 +0,0 @@
-module 0x42::mod1 {
```

Comment:
Why is this test file being removed?

Reply:
It doesn't make a sense anymore, because it tests for an error which this PR fixes.

These tests are also completely confusing since there is a ton of code in here but only one line which is tested. I believe those tests where created by using a larger example which run into the parse error and just c&p. We have the same code already in multiple other tests.